src/weather.cc

```diff
@@ -36,7 +36,224 @@
 #include "text_object.h"
 #include "weather.h"
 
-void print_weather(struct text_object *obj, char *p, unsigned int p_max_size) {
-  (void) obj;
-  strncpy(p ,"TODO...", p_max_size);
+static void curl_weather(void);
+static size_t read_weather_data_cb(char *, size_t, size_t, char *);
```

Comment:
`_owm_`? :face_with_head_bandage: ... and 10 more `_owm_`? :face_with_head_bandage: below.